src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -169,25 +170,33 @@ class ErgoWalletActor(ergoSettings: ErgoSettings) extends Actor with ScorexLoggi
           R5 -> StringConstant(description),
           R6 -> IntConstant(decimals)
         )
-        new ErgoBoxCandidate(0L, address.script, Seq(assetId -> amount), nonMandatoryRegisters)
+        val lockWithAddress = (addressOpt orElse trackedAddresses.headOption)
+          .getOrElse(throw new Exception("No address available for box locking"))
+        val candidateMock =
+          new ErgoBoxCandidate(100000L, lockWithAddress.script, Seq(assetId -> amount), nonMandatoryRegisters)
```

Comment:
This looks quite bad, and looks like we'll need it in more places. May be let's make some global function like 
`minimalErgoAmount(script, assets, nonMandatoryRegisters)` that will be used on box creation and also on block validation.